README.windows.rst

```diff
@@ -42,6 +42,9 @@ Current status
 The rados and rbd binaries and libs compile successfully and can be used on
 Windows, successfully connecting to the cluster and consuming pools.
 
+Ceph filesystems can be mounted using the ``ceph-dokan``, which requires the
+Dokany package to be installed.
```

Comment:
```suggestion
Ceph filesystems can be mounted using the ``ceph-dokan`` command, which
requires the Dokany package to be installed.
```

Reply:
Thanks for the suggestion, my bad for writing that in a hurry :).